mAppQuiz/mAppQuiz/App.xaml.cs

```diff
@@ -9,11 +9,18 @@ namespace mAppQuiz
 {
     public partial class App : Application
     {
+        public NavigationPage NavigationPage { get; private set; }
+
+
         public App()
         {
             InitializeComponent();
-
-            MainPage = new NavigationPage(new mAppQuiz.MainPage());
+            var menuPage = new MenuPage();
+            NavigationPage = new NavigationPage(new MainPage());
```

Comment:
Can we rename these to reflect what they represent instead of what type of object they are?